transport/api/src/lib.rs

```diff
@@ -358,11 +419,75 @@ where
         );
 
         // initiate the transport layer
-        let (ack_to_send_tx, ack_to_send_rx) = futures::channel::mpsc::unbounded::<(PeerId, Acknowledgement)>();
-        let (ack_received_tx, ack_received_rx) = futures::channel::mpsc::unbounded::<(PeerId, Acknowledgement)>();
+        let (ack_to_send_tx, ack_to_send_rx) = mpsc::unbounded::<(PeerId, Acknowledgement)>();
+        let (ack_received_tx, ack_received_rx) =
+            mpsc::channel::<(PeerId, Acknowledgement)>(MAXIMUM_ACK_INCOMING_BUFFER_SIZE);
+
+        let mixer_cfg = MixerConfig {
+            min_delay: std::time::Duration::from_millis(
+                std::env::var("HOPR_INTERNAL_MIXER_MINIMUM_DELAY_IN_MS")
+                    .map(|v| {
+                        v.trim()
+                            .parse::<u64>()
+                            .unwrap_or(hopr_transport_mixer::config::HOPR_MIXER_MINIMUM_DEFAULT_DELAY_IN_MS)
+                    })
+                    .unwrap_or(hopr_transport_mixer::config::HOPR_MIXER_MINIMUM_DEFAULT_DELAY_IN_MS),
+            ),
+            delay_range: std::time::Duration::from_millis(
+                std::env::var("HOPR_INTERNAL_MIXER_DELAY_RANGE_IN_MS")
+                    .map(|v| {
+                        v.trim()
+                            .parse::<u64>()
+                            .unwrap_or(hopr_transport_mixer::config::HOPR_MIXER_DEFAULT_DELAY_RANGE_IN_MS)
+                    })
+                    .unwrap_or(hopr_transport_mixer::config::HOPR_MIXER_DEFAULT_DELAY_RANGE_IN_MS),
+            ),
+            capacity: std::env::var("HOPR_INTERNAL_MIXER_CAPACITY")
+                .map(|v| {
+                    v.trim()
+                        .parse::<usize>()
+                        .unwrap_or(hopr_transport_mixer::config::HOPR_MIXER_CAPACITY)
+                })
+                .unwrap_or(hopr_transport_mixer::config::HOPR_MIXER_CAPACITY),
+            ..MixerConfig::default()
+        };
+        #[cfg(feature = "mixer-channel")]
```

Comment:
```suggestion
        #[cfg(all(feature = "mixer-channel", not(feature = "mixer-stream")))]
```

Reply:
The compilation would fail if both were specified. I can put a check that at least one is specified.